spec/decorators/user_decorator_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
```

Comment:
I think we can remove this, we already have the `,rspec` file

Reply:
good catch 💪